test/ctia/stores/es/store_test.clj

```diff
@@ -1,14 +1,27 @@
 (ns ctia.stores.es.store-test
-  (:require [ctia.store :as store]
+  (:require [clojure.set :as set]
+            [ctia.store :as store]
+            [clj-momo.test-helpers.core :as mth]
             [ctia.stores.es.store :as sut]
+            [ctia.entity.entities :as entities]
+            [ctia.entity.sighting :as sighting]
+            [ctim.examples.sightings :refer [sighting-minimal]]
+            [ctia.entity.sighting.schemas :refer [PartialStoredSighting StoredSighting]]
             [ctia.test-helpers.http :refer [app->APIHandlerServices]]
+            [ctia.test-helpers.fake-whoami-service :as whoami-helpers]
+            [ctia.test-helpers.auth :refer [all-capabilities]]
             [ctia.test-helpers.fixtures :as fixt]
             [ctia.test-helpers.core :as helpers]
             [ctim.examples.incidents :refer [incident-minimal]]
-            [clojure.test :refer [deftest testing is]]
+            [clojure.test :refer [deftest testing is use-fixtures]]
             [schema.core :as s]
             [schema-tools.core :as st]))
 
+(use-fixtures :once
+              mth/fixture-schema-validation
```

Comment:
At some point it would be good to get rid of `clj-momo`.
Please use [schema.test/validate-schemas](https://plumatic.github.io/schema/schema.test.html#var-validate-schemas)